src/components/Header/HeaderNotification/_mixins.scss

```diff
@@ -22,6 +22,34 @@
   box-sizing: border-box;
   justify-content: space-between;
 
+  // Inverse styling for string formatter's tooltip defintion:
```

Comment:
Hey @jendowns can you explain this part to me please. Thanks in advance!

Reply:
Yeah so there are a LOT of very specific styles that come from the `TooltipDefintion`, and this is me overriding a lot of that... (mostly this is due to theming, because we have themed the `Header` notification popover like a "light" themed component but the `TooltipDefinition` styles are anticipating a dark themed component --

----------

```
  .#{$namespace}string-formatter__tooltip.#{$prefix}--tooltip--definition
    .#{$prefix}--tooltip__trigger {
    color: currentColor;
    text-align: left;
    line-height: initial;
...
```

☝️ Without that, the color of the text in the `StringFormatter`, inside the `TooltipDefintion`, is *white* (unreadable), centered, and has weird line-height.

----------

```
    // Tooltip top variation arrow:
    &.#{$prefix}--tooltip__trigger--definition.#{$prefix}--tooltip--top.#{$prefix}--tooltip--align-start::before {
      border-color: $inverse-01 transparent transparent transparent;
    }

    // Tooltip bottom variation arrow:
    &.#{$prefix}--tooltip__trigger--definition.#{$prefix}--tooltip--bottom.#{$prefix}--tooltip--align-start::before {
      border-color: transparent transparent $inverse-01 transparent;
    }
```

☝️  Color of the tooltip's arrow is "light" themed so I need to make it inverse color & appear dark.

----------

```
    // Tooltip content:
    & + .#{$prefix}--assistive-text,
    &:focus
      + .#{$prefix}--assistive-text
      &:hover
      + .#{$prefix}--assistive-text {
      background-color: $inverse-01;
      color: $inverse-02;
    }
  }
```

☝️ For some reason the tooltip has different colors for `:hover` and for `:focus` states, hence this selector 😆  This rule inverts the colors as mentioned previously.

Reply:
@jendowns really appreciate the detailed explanation, thanks for clarifying ⭐️

Reply:
Yeah no worries, I hope it wasn't too much 😆 The selectors are so long it's almost hard to read, and the prettier formatting doesn't help... 🤔